src/control/cmd/dmg/pool.go

```diff
@@ -110,7 +115,57 @@ func (cmd *PoolCreateCmd) Execute(args []string) error {
 		return errors.Wrap(err, "parsing rank list")
 	}
 
-	if cmd.Size != "" {
+	switch {
+	case allFlagPattern.MatchString(cmd.Size):
+		if cmd.NumRanks > 0 {
+			return errIncompatFlags("all", "num-ranks")
+		}
+
+		// TODO (DAOS-9557) Update the protocol to allow filtering on ranks to use.  To
+		// implement this feature the procotol should be changed to define if a SCM
+		// namespace is associated with one rank or not. If yes, it should define with which
+		// rank the SCM namespace is associated.
+		if cmd.RankList != "" {
+			return errIncompatFlags("all", "ranks")
```

Comment:
Same note, please update the error.

Reply:
oops, sorry, i saw your comment after landing it. @knard-intel, please address in a new PR. Sorry @kjacque

Reply:
Whoops.

Reply:
fixed with PR https://github.com/daos-stack/daos/pull/7903